ARTICLE.md

```diff
@@ -1,59 +1,149 @@
-# Introduction to property based testing
-Mankind has evolved - perhaps it's time for how we test software to evolve as well?
+---
+title: From 10 to 1,000,000 test cases in under an hour: A beginner's guide to property-based testing 
+description: This guide walks you through the what and why of property-based testing, with practical use cases and examples along the way. 
+author: Fredrik Fornwall
+authorLink: https://dev.to/fornwall
+tags:
+  - testing
+  - tutorial
+  - beginners
+  - python
+---
```

Comment:
Open to any of this metadata can changing. But wanted to make sure we had the formatting in there for now.

Reply:
This formatting might change with the introduction of the new website 🤪 but at least all of the basics are there, so let's see.